ui-admin/src/forms/designer/questions/VisibilityFields.tsx

```diff
@@ -0,0 +1,61 @@
+import { unset } from 'lodash/fp'
+import React from 'react'
+
+import { Question } from '@juniper/ui-core'
+
+import { Checkbox } from 'components/forms/Checkbox'
+import { TextInput } from 'components/forms/TextInput'
+
+type VisibilityFieldsProps = {
+  disabled: boolean
+  question: Question
+  onChange: (newValue: Question) => void
+}
+
+/** Controls for editing other option for questions. */
```

Comment:
This looks like copy/paste from `OtherOptionFileds.tsx`. I wasn't reading exhaustively, so there may be others that I missed.

Reply:
Fixed in https://github.com/broadinstitute/juniper/pull/444/commits/f9dc871741c38f7339a97c2b554b84f7564eeb09.